exp-player/addon/components/exp-lookit-dialogue-page/component.js

```diff
@@ -708,17 +708,22 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
                  */
                 installPromise.then(() => {
                     this.send('setTimeEvent', 'recorderReady');
-                    this.set('recordingIsReady', true);
                     this.notifyPropertyChange('readyToStartAudio');
                 });
 
-                // Add event handlers on top of what the VideoRecordMixin normally does
+                // Add event handlers on top of what the VideoRecordMixin normally does - TODO: would ideally extend functionality of mixin handlers rather than replacing
```

Comment:
Yep, not a problem to repeat the code for now, just a note re: longer-term maintainability.